frontend/src/services/FlowService.ts

```diff
@@ -13,4 +13,19 @@ export async function getAllMeasures(): Promise<EcfmpMeasure[]> {
   }
 }
 
-export default { getAllMeasures };
+export async function setMeasureEnable(measureId: number, enabled: boolean): Promise<EcfmpMeasure[]> {
+  try {
+    const measures = await axios.patch<EcfmpMeasure[]>(
```

Comment:
API does currently return this type. This should be changed. In that case, the other function should be used to get all measures.